stdlib/3/xml/etree/ElementTree.pyi

```diff
@@ -2,126 +2,132 @@
+    tag = ... # type: _str_or_bytes
+    attrib = ... # type: Dict[_str_or_bytes, _str_or_bytes]
+    text = ... # type: Optional[_str_or_bytes]
+    tail = ... # type: Optional[_str_or_bytes]
+    def __init__(self, tag: Union[AnyStr, Callable[..., Element]], attrib: Dict[AnyStr, AnyStr]=..., **extra: Dict[str, AnyStr]) -> None: ...
+    def append(self, subelement: Element) -> None: ...
+    def clear(self) -> None: ...
+    def copy(self) -> Element: ...
+    def extend(self, elements: Sequence[Element]) -> None: ...
+    def find(self, path: str, namespaces: Dict[str, str]=...) -> Optional[Element]: ...
+    def findall(self, path: str, namespaces: Dict[str, str]=...) -> List[Element]: ...
+    def findtext(self, path: str, default: _T=..., namespaces: Dict[str, str]=...) -> Union[_T, str]: ...
+    def get(self, key: AnyStr, default: _T=...) -> Union[AnyStr, _T]: ...
+    def getchildren(self) -> List[Element]: ...
+    def getiterator(self, tag: Union[str, AnyStr]=...) -> List[Element]: ...
+    def insert(self, index: int, subelement: Element) -> None: ...
+    def items(self) -> ItemsView[AnyStr, AnyStr]: ...
+    def iter(self, tag: Union[str, AnyStr]=...) -> Generator[Element, None, None]: ...
+    def iterfind(self, path: str, namespaces: Dict[str, str]=...) -> List[Element]: ...
+    def itertext(self) -> Generator[str, None, None]: ...
+    def keys(self) -> KeysView[AnyStr]: ...
+    def makeelement(self, tag: _Ss, attrib: Dict[_Ss, _Ss]) -> Element: ...
+    def remove(self, subelement: Element) -> None: ...
+    def set(self, key: AnyStr, value: AnyStr) -> None: ...
+    def __bool__(self) -> bool: ...
+    def __delitem__(self, index: int) -> None: ...
+    def __getitem__(self, index) -> Element: ...
+    def __len__(self) -> int: ...
+    def __setitem__(self, index: int, element: Element) -> None: ...
+
+def SubElement(parent: Element, tag: AnyStr, attrib: Dict[AnyStr, AnyStr]=..., **extra: Dict[str, AnyStr]) -> Element: ...
+def Comment(text: _str_or_bytes=...) -> Element: ...
+def ProcessingInstruction(target: str, text: str=...) -> Element: ...
+
+PI = ... # type: Callable[..., Element]
 
 class QName:
-    text = ... # type: Any
-    def __init__(self, text_or_uri, tag=...) -> None: ...
-    def __hash__(self): ...
-    def __le__(self, other): ...
-    def __lt__(self, other): ...
-    def __ge__(self, other): ...
-    def __gt__(self, other): ...
-    def __eq__(self, other): ...
-    def __ne__(self, other): ...
+    text = ... # type: str
+    def __init__(self, text_or_uri: str, tag: str=...) -> None: ...
+    def __hash__(self) -> int: ...
+    def __le__(self, other: Union[str, QName]) -> bool: ...
+    def __lt__(self, other: Union[str, QName]) -> bool: ...
+    def __ge__(self, other: Union[str, QName]) -> bool: ...
+    def __gt__(self, other: Union[str, QName]) -> bool: ...
+    def __eq__(self, other: Union[str, QName]) -> bool: ...
+    def __ne__(self, other: Union[str, QName]) -> bool: ...
```

Comment:
I believe that __eq__ and __ne__ should accept any object, since you can compare anything with anything. Furthermore, I think you can avoid remove them altogether, since the superclass `object` already defines them.